### DISCUSSION TITLE
Regarding Loss vs epoch in MACE training

### DISCUSSION THREAD

**Discussion by SoumyaCYZ** [ACEsuit/mace#981] [Q&A] [answered]
Hi developers.  I'm observing that the loss first decreases, then there is a gradual increase in loss for a few molecules when I have used -----hidden_irreps =128x0e. But when I reduced fix -- hidden_irreps=number of atoms present in molecule, I  have seen the loss gets saturated using the less epoch (epoch=100), and the potential is quite stable.  Is there a situation of overfitting if we increase the number of epochs at higher hidden_irreps?

**Comment by SoumyaCYZ** [comment]
@RokasEl @ilyes319 Sir, can you please help?

**Reply by ilyes319** [reply, marked as answer]
mace automatically saves the model with the lowest validation loss, so even if it is going up, it will be fine.
I don't understand that `-- hidden_irreps=number of atoms present in molecule`. the --hidden_irreps controls the size of the model. The recommended size is the size of a medium model --hidden_irreps="128x0e + 128x1o"?
